src/clipctl.c

```diff
@@ -89,11 +89,30 @@ int main(int argc, char *argv[]) {
     expect(kill(pid, want_enable ? SIGUSR2 : SIGUSR1) == 0);
     dbg("Sent signal to pid %d\n", pid);
 
+    unsigned int delay_ms = 1;
+    unsigned int total_wait_ms = 0;
+    const unsigned int max_wait_ms = 1000;
+
     for (size_t retries = 0; retries < MAX_STATE_RETRIES; ++retries) {
```

Comment:
`MAX_STATE_RETRIES` seems redundant now. Especially since (if my math is right) it will never be reached due to `total_wait_ms` reaching 1023 after 10 iterations.

I suggest gutting it and changing the loop condition like so:

```suggestion
    while (total_wait_ms < max_wait_ms) {
```

Reply:
oh yes, that was silly to leave in. thanks